python/smqtk/algorithms/relevancy_index/libsvm_hik.py

```diff
@@ -84,6 +84,13 @@ def __init__(self, descr_cache_filepath=None, autoneg_select_ratio=1,
             of None means to use all available cores.
         :type cores: int | None
 
+        :param svm_model: The SVM model used.
+        :type svm_model | None
+
+        :param use_libsvm: Whether we wish to use LIBSVM to do the scaling or
```

Comment:
This is not currently a parameter to the constructor.  This should be so it may be a configurable option.

The type specification should also be of the form: ``:type use_libsvm: bool``